storage/feed.go

```diff
@@ -255,6 +257,42 @@ func (s *Storage) fetchFeedsWithCounters(query string, args ...interface{}) (mod
 	return feeds, nil
 }
 
+// FeedSchedulerParameters returns the parameters used for the scheduler.
+func (s *Storage) FeedSchedulerParameters(userID, feedID int64) (int, error) {
+	scheduler := strings.ToLower(config.Opts.PollingScheduler())
+	if scheduler != model.SchedulerEntryCountBased {
+		return 0, nil
+	}
+	var feed model.Feed
+	var weeklyCount int
+
+	query := `
+		SELECT
+			f.id,
+			(SELECT count(*) FROM entries 
+				WHERE entries.feed_id=f.id AND 
+				(entries.published_at Between (now() - interval '7 days') and now()) 
+				) as weekly_count
+		FROM feeds f
+		WHERE
+			f.user_id=$1 AND f.id=$2
+	`
+
+	err := s.db.QueryRow(query, userID, feedID).Scan(
+		&feed.ID,
```

Comment:
`feed.ID` and `feed` don't seems to be necessary because they are not used. Only the weekly count is returned by the function.